apps/web/src/app/events/components/AttendanceCard/AttendanceCard.tsx

```diff
@@ -15,13 +15,32 @@ import { TicketButton } from "./TicketButton"
 import { ViewAttendeesDialogButton } from "./ViewAttendeesButton"
 import { useDeregisterMutation, useRegisterMutation } from "./mutations"
 
+const getQueuePosition = (
+  attendee: Attendee | undefined,
+  attendees: Attendee[] | undefined,
+  attendablePool: AttendancePool | undefined
+) => {
```

Comment:
It should probably not be possible to return undefined, null, or number from this. Maybe return `null` instead of undefined in the first branch